tests/OSPSuite.SimModel.Tests/src/SimulationSpecs.cpp

```diff
@@ -611,6 +614,7 @@ namespace SimulationTests
 
     public:
         [TestAttribute]
+        [Ignore("")]
```

Comment:
That's useful message :)

Reply:
lol. with 99% probabilty this functionality will be removed soon, so the tests will be removed as well